README.md

```diff
@@ -415,45 +403,111 @@ This field SHOULD NOT be used to sign arbitrary data, such as signature challeng
 
 ### 3.2.4 Facts
 
-The OPTIONAL `fct` field contains arbitrary facts and proofs of knowledge. The enclosed data MUST be self-evident and externally verifiable. It MAY include information such as hash preimages, server challenges, a Merkle proof, dictionary data, etc.
+The OPTIONAL `fct` field contains a map of arbitrary facts and proofs of knowledge. The enclosed data MUST be self-evident and externally verifiable. It MAY include information such as hash preimages, server challenges, a Merkle proof, dictionary data, etc.
 
 #### Examples
 
 ``` json
-"fct": [
-  {
-    "challenge": "abcdef",
-    "from": "example.com"
-  },
-  {
-    "sha3_256": "B94D27B9934D3E08A52E52D7DA7DABFAC484EFE37A5380EE9088F7ACE2EFCDE9",
-    "msg": "hello world"
+{
+  "fct": {
+    "challenges": {
+      "example.com": "abcdef",
+      "another.example.net": "12345"
+    },
+    "sha3_256": {
+      "B94D27B9934D3E08A52E52D7DA7DABFAC484EFE37A5380EE9088F7ACE2EFCDE9": "hello world"
+    }
   }
-]
+}
 ```
 
-### 3.2.5 Attenuation
+### 3.2.5 Capabilities & Attenuation
 
-The attenuations (i.e. UCAN output, or "caveats") MUST be an array of heterogeneous capabilities (defined below). This array is REQUIRED but MAY be empty.
+Capabilities MUST be presented as a map. This map is REQUIRED but MAY be empty.
 
 This array MUST contain some or none of the following:
-1. A strict subset (attenuation) of the capability scope from the `prf` field
-2. Capabilities composed from multiple proofs (see rights amplification [§6.4](#64-rights-amplification))
+1. A strict subset (attenuation) of the capability authority from the `prf` field
+2. Capabilities composed from multiple proofs (see [rights amplification])
 3. Capabilities originated by the `iss` DID (i.e. by parenthood)
 
-This array also includes time ranges and the proofs that start latest and end soonest from the lower and upper time bounds of the range.
+#### 3.2.5.1 Anatomy
 
-### 3.2.7 Proof of Delegation
+The anatomy of a capability MUST be given as a mapping of resource URI to abilities to array of caveats or the wildcard `*`:
 
-Attenuations MUST be satisfied by matching the attenuated capability to a proof in the `prf` array ([§3.2.7.1](#3271-prf-field))
+```
+{ $RESOURCE: { $ABILITY: "*" } }
+{ $RESOURCE: { $ABILITY: [ ...$CAVEATS ] } }
```

Comment:
This may be controversial. I promise that there's a very good, very logical reason. See below.